system/security/shared/caching.cpp

```diff
@@ -556,7 +556,7 @@ bool CPermissionsCache::queryPermsManagedFileScope(ISecUser& sec_user, const cha
     if (!fullScope || !*fullScope)
     {
         *accessFlags = queryDefaultPermission(sec_user);
-        WARNLOG("FileScope unspecified for %s, applying default permissions %s(%d), took %dms", sec_user.getName(), getSecAccessFlagName(*accessFlags), *accessFlags,  msTick()-start);
+        OWARNLOG("FileScope missing, using root for %s, applying default permissions %s(%d), took %dms", sec_user.getName(), getSecAccessFlagName(*accessFlags), *accessFlags,  msTick()-start);
```

Comment:
It isn't that the file scope is missing, it is that the file scope is empty.  Having said that this should be uncommon that it doesn't really matter.

Reply:
Changed the message to say empty.